Update Advisor TOC.md with link to CLI documentation

mattwag opened the pull request:
Advisor recently added CLI support.  Updating the Reference section of the TOC to include a link to the CLI documentation.  I also changed the name of the 'REST' link to 'REST API' to be a bit more clear.

azurecla commented:
Hi __@gopitk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

MonicaRush commented:
@bradsev 
Brad -- Hi -- Would you like to review these changes and let me know if they're OK? Thanks -- Monica

bradsev commented:
I have reviewed. Please accept these edits. Thx Monica. -Brad

gopitk commented:
Monica/Brad - Looks like this content never got pushed to prod. Is that
automatic or does someobe have to do anything to push contributed content?
There are two other updates I submitted and were merged late last week.

Thanks,
Gopi Kumar
Sr PM, IMML


On Fri, Jul 24, 2015 at 5:33 PM, Monica Rush <notifications@github.com>
wrote:

> Merged #4073 <https://github.com/Azure/azure-content/pull/4073>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-content/pull/4073#event-365059164>.
>

MonicaRush commented:
@gopitk @bradsev 
Gopi -- I don't know why this article was not published. Can you please check again after 10:00 am? If it is not live let me know and I'll look into it further. Sorry for the delay -- Monica

mattwag committed:
Update TOC.md

Advisor recently added CLI support.  Updating the Reference section of the TOC to include a link to the CLI documentation.  I also changed the name of the 'REST' link to 'REST API' to be a bit more clear.
